crates/bitcoin-da/src/spec/utxo.rs

```diff
@@ -1,7 +1,7 @@
 use bitcoin::Txid;
 use serde::{Deserialize, Serialize};
 
-#[derive(Clone, Debug, PartialEq, Serialize)]
+#[derive(Clone, Debug, PartialEq, Eq, Serialize, Hash)]
```

Comment:
Why is derive Hash is added?

If we need it, we can use txid and vout as it's unique identifier.

Reply:
Oh, I was playing with hashmaps. Will fix